embedded-google-pubsub/src/test/java/com/playtika/test/pubsub/EmbeddedPubsubBootstrapConfigurationTest.java

```diff
@@ -94,4 +99,19 @@ private void hasDependsOn(String beanName) {
                 .isNotEmpty()
                 .contains(BEAN_NAME_EMBEDDED_GOOGLE_PUBSUB);
     }
+
+    @Test
+    public void shouldCreateManagedChannelWithCorrectIpAndPort() {
+        GenericContainer pubsub = mock(GenericContainer.class);
```

Comment:
this test is out of scope of this test class,
means this test class basically contain functional test, but newly added test is just simple unit-test
as a result I cannot run this test without starting whole context, but it's not really required here.

Reply:
can't we just get ```ManagedChannel``` from context and get data we are interesting in order to verify it?

Reply:
Hi, I understand that is not the assertions and neither the best place for the test, if we decide to go with this solution I can move it to a simple class without spring context. Here are all the possible solutions that came to my mind.

Regarding getting the data from the ManagedChannel, **that was my first attempt**, so we could assert like:
assertThat(managedChannel.getHost(), is("valueHere"))

but unfortunately **neither the ManagedChannel or ManagedChannelBuilder expose the field.**

Then I thought about passing the ManagedChannelBuilder as a dependency and try to assert on the parameter passed to it with an argumentCaptor. 

However the method that we use to create the managedChannel is static, so to do that assertion we will need a heavier testing framework, as PowerMock(ito). Please check the link: https://github.com/powermock/powermock/wiki/Mockito#mocking-static-method

I felt it would be just too much to add that test dependency to make sure we are attributing the correct variable, however if that is ok for you we can add it.

In your opinion which path should we choose?

Regards

Reply:
Hi @tdanylchuk any decision on this?

Reply:
this test, basically tests almost nothing, dunno whether we even need it... too bad that we cannot find better way to test it. For now, im okay with deleting this test at all, if issue occur, we would need to find better way to test.

By now, just remove this test, and let's merge...

Reply:
@tdanylchuk I totally feel it man. I also wish they could provide a better api. But with all those static methods is hard to test :/